Source/CBLRemoteRequest.m

```diff
@@ -176,7 +176,9 @@ - (NSMutableDictionary*) statusInfo {
 
 - (void) respondWithResult: (id)result error: (NSError*)error {
     Assert(result || error);
-    _onCompletion(result, error);
+
+    CBLRemoteRequestCompletionBlock onCompletion = _onCompletion;   // keep block alive till return
```

Comment:
It could, but that wouldn't change anything since its type is `CBLRemoteRequestCompletionBlock`.

(I fixed this method first, and then after searching the codebase for more I came up with the `__typeof` shortcut to make it easier to copy and paste the fix.)